files/common/lib/modprobe.d/10-zfs.conf

```diff
@@ -137,3 +137,12 @@ options zfs zfs_deadman_synctime_ms=60000
 # enabled, thus we disable it here until we can get that fixed.
 #
 options zfs zap_shrink_enabled=0
+
+#
+# On object storage, block cloning causes a VM panic on small instances.  The
+# problem is that we do an allocation proportional to the size of the vdev,
+# which is huge for object storage. We could try to change the block cloning
+# code to behave better in the presence of large vdevs, but for now, we should
+# just disable the feature.
+#
+options zfs zfs_bclone_enabled=0
```

Comment:
Do we want to disable this feature on `develop` or should this PR target `os-upgrade`? I don't know enough about the feature to make this decision but lgtm otherwise.

Reply:
We might as well target develop, because if this code path somehow got triggered by some other utility on 20.04 (not impossible), we would also hit the hang there.